src/searchForm/baseStyles.tsx

```diff
@@ -32,9 +32,9 @@ export const sectionBaseStyle = `
 
 export const autoCompleteBaseStyle = `
   .kirk-textField-wrapper {
-    background: transparent;
-    border-radius: 0;
-    border: 0;
+    background: transparent !important;
```

Comment:
can't we avoid using this class on this component then?
 it is really why css-in-js should be used instead of nested classes :(

Reply:
We can and this is what is desribed in the PR description
> Use !important as quick fix, better fix would be not to use nested selectors for styles override, it breaks components isolation and introduce these kinds of errors.

Reply:
still don't get it

Reply:
We're doing a quick fix for a prod issue, it's probably not the right time to change everything :)